packages/suite/src/utils/suite/stake.ts

```diff
@@ -412,8 +412,8 @@ export const prepareClaimEthTx = async ({
     }
 };
 
-interface GetStakeTxGasLimitParams {
-    ethereumStakeType: StakeType | undefined;
+export interface GetStakeTxGasLimitParams {
+    ethereumStakeType: StakeType | undefined | 'unclaim';
```

Comment:
What is `unclaim`?

Reply:
added for testing. Removed https://github.com/trezor/trezor-suite/pull/13443/commits/080d92b42ab54354479ade7492ad05dd986a182f